clusters/hosted-mgmt/hive/pools/rhtap/rhtap-ocp-4-14-0-amd64-aws-us-east-2_clusterpool.yaml

```diff
@@ -33,7 +33,7 @@ spec:
   pullSecretRef:
     name: pull-secret
   runningCount: 1
-  size: 10
+  size: 5
```

Comment:
FYI the load on infra-deployments side is much bigger than on e2e-tests side, e.g. in June we had [~504 jobs runs for infra-deployments](https://frontend-quality-dashboard.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/reports/test?team=Konflux&organization=redhat-appstudio&repository=infra-deployments&job_type=presubmit&start=2024-06-01+00%3A56%3A00&end=2024-06-30+21%3A56%3A00&job_name=pull-ci-redhat-appstudio-infra-deployments-main-appstudio-e2e-tests) vs. [122 e2e job runs in e2e-tests repo](https://frontend-quality-dashboard.apps.stone-stg-rh01.l2vh.p1.openshiftapps.com/reports/test?team=Konflux&organization=konflux-ci&repository=e2e-tests&job_type=presubmit&start=2024-06-01+00%3A56%3A00&end=2024-06-30+21%3A56%3A00&job_name=pull-ci-konflux-ci-e2e-tests-main-redhat-appstudio-e2e)

So I think the cluster pool size should go down for ~1/4, i.e.  from 10 to 7, but I'm ok with keeping it as is and increase the pool size if we hear complains  ;)